src/master-playlist-controller.js

```diff
@@ -370,7 +336,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     // Pause any alternative audio
     if (this.audioPlaylistLoader_) {
       this.audioPlaylistLoader_.pause();
-      this.audioPlaylistLoader_ = null;
+      this.audioPlaylistLoader_.dispose();
```

Comment:
the problem comes in when we set an `audioPlaylistLoader_` to null. which sets an entry in `audioPlaylistLoaders_` to null. Then when we go to switch back to the audio a second time there is no `PlaylistLoader` to use for that track. Maybe we could not create all the playlist loaders at the start? and only create them when we use them?